## Description: Makes images in GtkStatusIcon use fallbacks by default if the property is available
## Origin/Author: Ted Gould <ted@canonical.com>
## Bug: http://launchpad.net/bugs/729150
Index: b/gtk/gtkstatusicon.c
===================================================================
--- a/gtk/gtkstatusicon.c
+++ b/gtk/gtkstatusicon.c
@@ -870,6 +870,11 @@
 		    	    G_CALLBACK (gtk_status_icon_screen_changed), status_icon);
   priv->image = gtk_image_new ();
   gtk_widget_set_can_focus (priv->image, TRUE);
+
+  if (g_object_class_find_property(G_OBJECT_CLASS(GTK_IMAGE_GET_CLASS(priv->image)), "use-fallback") != NULL) {
+    g_object_set(G_OBJECT(priv->image), "use-fallback", TRUE, NULL);
+  }
+
   gtk_container_add (GTK_CONTAINER (priv->tray_icon), priv->image);
   gtk_widget_show (priv->image);
 
